src/sonic-yang-models/setup.py

```diff
@@ -86,6 +86,7 @@ def run(self):
                          './yang-models/sonic-bgp-peergroup.yang',
                          './yang-models/sonic-bgp-peerrange.yang',
                          './yang-models/sonic-bgp-allowed-prefix.yang',
+                         './yang-models/sonic-bgp-voq-chassis-neighbor.yang',
```

Comment:
Need to add this file to cvlyang-models

Reply:
Hi @ganglyu, currently the `BGP_VOQ_CHASSIS_NEIGHBOR` and `BGP_INTERNAL_NEIGHBOR` tables are used by 'bgpcfgd` I am not sure if the sonic-mgmt-framework or `frrcfgd` will use these tables. Hence, I did not add them to the cvlyang-models

@venkatmahalingam, can you please confirm if it is ok to add the yang models for   `BGP_VOQ_CHASSIS_NEIGHBOR` and `BGP_INTERNAL_NEIGHBOR` to cvl models.

Reply:
Yes, that's correct, we dont use these models in frrcfgd.